Assets/Scripts/Item/ItemObjects.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+public enum ItemType {
+    Item_Jump = 0,
+    Item_Climb = 1,
+    Item_left = 2,
+    Item_Rush = 3,
+    Item_Throw = 4,
+    Item_SlotAdd = 5,
+}
+public class ItemObjects : MonoBehaviour {
+    [SerializeField] ItemType Type;//itemType
+    [SerializeField] Sprite[] itemSprite;//SettingSprites
+    [SerializeField]SpriteRenderer SrChild;//getChildSprite
```

Comment:
Is the `SpriteRenderer` attached to `ItemObjects` game object, or rendered in its child?